src/renderer/services/docker/chain.ts

```diff
@@ -44,7 +44,16 @@ export class BeaconChain extends Container {
         for (let i = 0; i < savedNodes.length; i++) {
             savedNodes[i].nodes.map((node) => {
                 if (node.localDockerId) {
-                    promises.push(Container.startStoppedContainer(node.localDockerId));
+                    const promise = new Promise(async(resolve, reject) => {
+                        const image = await Container.getImage(node.localDockerId);
+                        if (image) {
+                            await BeaconChain.loadStoppedChain(node.localDockerId, image);
+                        } else {
+                            return reject("Image not found.");
+                        }
+                        resolve();
+                    });
```

Comment:
Why don't you make function out this rather than some confusing promise ?

Reply:
this wasn't ready for rreview :P